src/main/java/com/iexec/core/task/TaskUpdateManager.java

```diff
@@ -251,7 +251,26 @@ void initializing2Initialized(Task task) {
 
     void initialized2Running(Task task) {
         String chainTaskId = task.getChainTaskId();
-        boolean condition1 = replicatesService.getNbReplicatesWithCurrentStatus(chainTaskId, ReplicateStatus.STARTING, ReplicateStatus.COMPUTED) > 0;
+
+        // We explicitly exclude START_FAILED as it could denote some serious issues
+        // The task should not transition to `RUNNING` in this case.
+        final int nbReplicatesContainingStartingStatus = replicatesService.getNbReplicatesWithCurrentStatus(
```

Comment:
The worker replicate could be in `WORKER_LOST` or `RECOVERING` as a current status after being in `DATA_DOWNLOADING` for example.

Reply:
You might mix getNbReplicatesWithCurrentStatus and getLastRelevantStatus in a dedicated method.

Reply:
That's what I was working on :)

Reply:
Fixed in https://github.com/iExecBlockchainComputing/iexec-core/pull/475/commits/56e79014a55d775b05012f4fa0e7362ddc2a0ab6

Reply:
Thank you for the fix. All good for me then.